mep/books/models.py

```diff
@@ -545,6 +555,9 @@ class CreatorType(Named, Notable):
     '''Type of creator role a person can have in relation to a work;
     author, editor, translator, etc.'''
 
+    order = models.PositiveSmallIntegerField(
+        help_text='order in which creators will be listed')
```

Comment:
Maybe revise to specify _types_ of creators?  Only suggesting this because I'm adding an order to creators in my branch and want to avoid confusion as much as possible.